libs/pages/services/src/lib/feature/page-job-create-feature/step-dockerfile-feature/step-dockerfile-feature.tsx

```diff
@@ -81,7 +89,7 @@ export function StepDockerfileFeature() {
                 Back
               </Button>
               <div className="flex gap-3">
-                <Button type="submit" size="lg" disabled={isLoadingCheckDockerfile}>
+                <Button type="submit" size="lg" disabled={isLoadingCheckDockerfile || !isValid}>
```

Comment:
Missing `loading` state to check if dockerfile is valid

Reply:
Fixed with https://github.com/Qovery/console/pull/1459/commits/bd2624514654845e05ebb145d89ec34f2ebd9036